.github/renovate.json

```diff
@@ -35,14 +36,131 @@
       "matchUpdateTypes": ["minor", "patch"]
     },
     {
-      "groupName": "npm minor",
-      "matchManagers": ["npm"],
-      "matchUpdateTypes": ["minor", "patch"]
+      "matchPackageNames": ["DnsClient", "Quartz"],
+      "description": "Admin Console owned dependencies",
+      "reviewers": ["team:team-admin-console-dev"]
     },
     {
-      "groupName": "nuget minor",
-      "matchManagers": ["nuget"],
-      "matchUpdateTypes": ["minor", "patch"]
```

Comment:
So you also want to get rid of NPM and NuGet grouping?

Reply:
Yes, it makes it difficult to get an overview of the impact.